src/app/clusters/color-control-server/color-control-server.cpp

```diff
@@ -267,6 +279,59 @@ static uint8_t readLevelControlCurrentLevel(EndpointId endpoint)
     return currentLevel;
 }
 
+static uint8_t readColorLoopActiveAttribute(EndpointId endpoint)
+{
+    uint8_t isColorControleActive;
+    EmberAfStatus status;
+
+    status = emberAfReadServerAttribute(endpoint, ZCL_COLOR_CONTROL_CLUSTER_ID, ZCL_COLOR_CONTROL_COLOR_LOOP_ACTIVE_ATTRIBUTE_ID,
+                                        reinterpret_cast<uint8_t *>(&isColorControleActive), sizeof(uint8_t));
+
+    if (status != EMBER_ZCL_STATUS_SUCCESS)
+    {
+        isColorControleActive = COLOR_LOOP_INACTIVE;
+    }
+
+    return isColorControleActive;
+}
+
+static uint16_t readColorLoopStoredEnhancedHueAttribute(EndpointId endpoint)
+{
+    uint16_t storedEnhancedHue;
+    emberAfReadServerAttribute(endpoint, ZCL_COLOR_CONTROL_CLUSTER_ID,
+                               ZCL_COLOR_CONTROL_COLOR_LOOP_STORED_ENHANCED_HUE_ATTRIBUTE_ID,
+                               reinterpret_cast<uint8_t *>(&storedEnhancedHue), sizeof(uint16_t));
+
+    return storedEnhancedHue;
+}
+
+static uint16_t readColorLoopStartEnhancedHueAttribute(EndpointId endpoint)
```

Comment:
As Boris mentionned we can delete all those read functions and use `Accessors.h`